_episodes/09-SWAN.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Using data in CloudStor"
+teaching: 5
+exercises: 5
+questions:
+- "How do you work in Jupyter Notebooks with data stored in CloudStor?"
+objectives:
+- "Use data stored in Cloudstor in Jupyter Notebooks"
+activity:
+- "Bring a dataset into a notebook"
+keypoints:
+ - "This is where the fun begins!"
+---
+
+# Using data in CloudStor
+
+CloudStor's SWAN (Service for Web-based Analysis) presents the [Jupyter Lab](https://towardsdatascience.com/jupyter-lab-evolution-of-the-jupyter-notebook-5297cacde6b) environment.
```

Comment:
I'm not sure whether we want to link to `towardsdatascience.com` which after a few visits per month they ask you to pay for more.

Reply:
OK I will look for a better article. Thanks.

Reply:
OK - will point to the JupyterLab overview: https://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html